x-pack/solutions/security/packages/kbn-securitysolution-exception-list-components/src/exception_item_card/meta/index.tsx

```diff
@@ -104,17 +104,19 @@ export const ExceptionItemCardMetaInfo = memo<ExceptionItemCardMetaInfoProps>(
             )}
           </>
         )}
-        <EuiFlexItem>
-          <HeaderMenu
-            emptyButton
-            useCustomActions
-            iconType="list"
-            actions={referencedLinks}
-            disableActions={false}
-            text={i18n.AFFECTED_RULES(rules.length)}
-            dataTestSubj={dataTestSubj}
-          />
-        </EuiFlexItem>
+        {referencedLinks != null && (
```

Comment:
Is this change somehow related to RBAC?

Reply:
This fixes a bug the new RBAC role possibilities revealed when a user had access to siem endpoint exceptions but not rules, the endpoint item cards would render this weird empty popover button